sql/derived_tables/consortial_view.sql

```diff
@@ -25,13 +26,16 @@ FROM
             reshare_rs.directory_entry de2
         WHERE
             de2.de_parent IS NULL
-            AND de2.de_status_fk IS NOT NULL) AS names ON prr.__origin = names.__origin
+            AND de2.de_status_fk IS NOT NULL
+            AND de2.de_name NOT IN ('Bobst Circulation Desk', 'South Charleston Campus Collection')) AS names ON prr.__origin = names.__origin
```

Comment:
I think we need to find a way to do this without including PALCI-specific locations in the community queries.  Would it be possible to create a PALCI-specific derived table that selects this table (`consortial_view`) and filters the results?  If so, that derived table could be stored in PALCI's own repository.

Reply:
@nassibnassar I think removing the "filter" and putting it elsewhere is a fine idea, I just have no clue about how to go about that from the sense of where is this PALCI-specific repository, who creates it, how does it stay up-to-date with further changes, how is their actual reporting interface configured, etc..  Currently this "filter" is in four derived tables, which are then referenced in various report queries.  I can handle writing the SQL, but everything else is outside of my access/expertise/etc..  Also, the reason we need this "filter" is because of an issue in ReShare, and I have no sense of when it's getting fixed (I think it's related to [PR-1322](https://openlibraryfoundation.atlassian.net/browse/PR-1322)).  If it's fixed soon, all of this would be unnecessary.

Reply:
Great, thanks for the background.  I will look into all of this.  (We do often have library-specific repositories for LDP queries.)  In the meantime, if you would like to make a note of the PALCI-specific changes and then remove them from this PR, I can continue with testing the PR.

Reply:
I removed those filters.